### PR TITLE
COMMERCE-5406 Refactor - sort agnostic implementation of DispatchTriggerMetadata + COMMERCE-6703 disable scheduler activation form and warn user about problem

### PR DESCRIPTION
**Pull request by igorbeslic**

### CODE REVIEW

**Comment by igorbeslic** [comment]
ci:test:sf

**Comment by igorbeslic** [comment]
ci:test:sf

**Comment by igorbeslic** [comment]
ci:forward

File: modules/apps/dispatch/dispatch-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -24,6 +24,7 @@ runtime=Runtime
 scheduled-jobs=Scheduled Jobs
 single-node=Single Node
 task-execution-cluster-mode=Task Execution Cluster Mode
+task-executor-is-not-ready=Task executor is not ready
```
Comment (dsitu) [inline modules/apps/dispatch/dispatch-lang/src/main/resources/content/Language.properties:27]: Missing period here.

```suggestion
task-executor-is-not-ready=Task executor is not ready.
```

**Comment by igorbeslic** [comment]
ci:test:sf

**Comment by igorbeslic** [comment]
ci:stop

**Comment by igorbeslic** [comment]
ci:forward

**Comment by igorbeslic** [comment]
ci:stop

**Pull request closed without merging**